tests/AllenCahn/input

```diff
@@ -9,6 +9,11 @@
 #@  dim    = 2
 #@  check  = false
 #@  args   = stop_time=1.0
+#@
+#@  [2D-random.ic-test]
+#@  dim    = 2
+#@  check  = true
+#@  args   = type = random
```

Comment:
@mongole584 I do not think this will run successfully, since the different initial conditions will cause there to be different results. Recommend you add a different check-file and update the test script accordingly.